src/Sulu/Bundle/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -30,6 +30,7 @@ public function getConfigTreeBuilder()
         $treeBuilder->root('sulu_admin')
             ->children()
                 ->scalarNode('name')->defaultValue('Sulu Admin')->end()
+                ->scalarNode('email')->defaultValue('hikari.sulu@su.lu')->end()
```

Comment:
I would not put a default value here, either it is a required value, which has to be set or it is optional.